build-tests/api-extractor-scenarios/etc/exportImportStarAs2/api-extractor-scenarios.api.json

```diff
@@ -197,7 +197,7 @@
                 {
                   "kind": "Reference",
                   "text": "ForgottenClass",
-                  "canonicalReference": "api-extractor-scenarios!~ForgottenClass:class"
+                  "canonicalReference": "api-extractor-scenarios!ForgottenClass:class"
```

Comment:
This change may look unexpected because it's to a reference token, and this PR doesn't touch `DeclarationReferenceGenerator`. It's in fact unrelated to this PR entirely - and is a consequence of a dependency between the two PRs that were merged last week. We should've ran `rush rebuild` between the two PRs.